From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Tue, 9 Feb 2021 17:09:48 +0000
Subject: [PATCH] Add missing dep for libparted-fs-resize.pc

The libparted-fs-resize library references symbols from libparted, so
the latter is a dependency of the former.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 libparted-fs-resize.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libparted-fs-resize.pc.in b/libparted-fs-resize.pc.in
index ed9b3d6..3194600 100644
--- a/libparted-fs-resize.pc.in
+++ b/libparted-fs-resize.pc.in
@@ -6,5 +6,6 @@ includedir=@includedir@
 Name: libparted-fs-resize
 Description: The GNU Parted filesystem resize shared library
 Version: @VERSION@
+Requires: libparted
 Libs: -L${libdir} -lparted-fs-resize
 Cflags: -I${includedir}
